src/toil/jobStores/abstractJobStore.py

```diff
@@ -292,13 +297,13 @@ def _writeToUrl(cls, readable, url):
 
         Refer to AbstractJobStore.importFile documentation for currently supported URL schemes.
 
-        :param str url: URL that points to a file or object in the storage mechanism of a
+        :param urlparse.ParseResult url: URL that points to a file or object in the storage mechanism of a
                 supported URL scheme e.g. a blob in an Azure Blob Storage container.
         :param readable: a readable stream
         """
         raise NotImplementedError()
 
-    @abstractclassmethod
+    @classmethod
     def _supportsUrl(cls, url):
         """
         Returns True if the job store supports the URL's scheme.
```

Comment:
`… for importing from or exporting to.`

We also need to discuss how to represent the case where a URL is supported for import only.